examples/training/classification/basic_imagenet_training.py

```diff
@@ -196,7 +203,7 @@ def augment(img, label):
 """
 
 
-metrics = [metrics.CategoricalAccuracy()]
+metrics = ["accuracy"]
```

Comment:
I think Francois wanted to use a specific metric instance right?  lets keep it in line with the other ones

Reply:
Yes, but it turns out that this throws an error because if you do it with a real object you have to do it in the same strategy scope as you instantiate the model.

I'll just update this to re-open the scope when instantiating this.